assignments/callbacks.js

```diff
@@ -24,25 +24,51 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
```

Comment:
all of your callbacks look good to me. Seems like you're really getting the point of the lesson and how to use callbacks correctly.